src/property-filter/styles.scss

```diff
@@ -83,6 +85,91 @@
   }
 }
 
+.token-editor-grouped {
+  display: flex;
+  flex-direction: column;
+  gap: awsui.$space-s;
+  user-select: text;
+  margin-block: awsui.$space-xxs;
+  margin-inline: awsui.$space-xxs;
+
+  &-form {
```

Comment:
I'm not sure if we have any agreements on that, but from my perspective writing css classes this way makes it difficult to find and debug them in future.

WDYT?

Reply:
I am fine with either keeping that style or banning it, but there are currently 300+ usages of this style in the code. I don't see it as a big concern as hovering over that line in most editors would show the complete class name.